scripts/release.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
```

Comment:
@levithomason there are no licensing issues, right? Just wanted to be sure that we have permission to copy that script into our code-base without also copying in its Apache license.